src/CommunityToolkit.Maui.MediaElement/MediaSource/MediaSource.shared.cs

```diff
@@ -9,6 +9,7 @@ namespace CommunityToolkit.Maui.Views;
 [TypeConverter(typeof(MediaSourceConverter))]
 public abstract class MediaSource : Element
 {
+	static bool IsAndroid => DeviceInfo.Current.Platform == DevicePlatform.Android;
```

Comment:
can this be cached? Using the `=>` expression will make this be evaluated everytime and I doubt there's a scenario where this value will change.

```suggestion
	static bool IsAndroid { get; } =  DeviceInfo.Current.Platform == DevicePlatform.Android;
```

Reply:
I updated it with suggested change